src/patternfly/components/Content/styles.scss

```diff
@@ -40,143 +139,111 @@
   h5,
   h6 {
     margin: 0;
-    font-weight: $global--FontWeight--normal;
-    line-height: $global--LineHeight;
+  }
+
+  .pf-is-hero {
```

Comment:
Content is meant for raw HTML, correct?

Reply:
you are right, it isn't... it should be `pf-content__hero-title` right?